l1-contracts/Dockerfile

```diff
@@ -17,7 +17,9 @@ RUN forge install --no-commit \
   https://github.com/openzeppelin/openzeppelin-contracts
 ENV MAINNET_RPC_URL='https://mainnet.infura.io/v3/9928b52099854248b3a096be07a6b23c'
 
+COPY . .
```

Comment:
I dont think moving this copy around will have an effect since its all happening within the one layer anyway?

Reply:
Copying after doing the installations will make us cache the installation so we don't have to re-install all the time. Madkes it easier to rerun